benefits/urls.py

```diff
@@ -25,6 +25,7 @@
     path("enrollment/", include("benefits.enrollment.urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("oauth/", include("benefits.oauth.urls")),
+    path("in_person/", include("benefits.in_person.urls")),
```

Comment:
Can we make this top level URL `in-person/` instead? I think it looks a little more natural in a URL than underscores.

Reply:
Ah, disregard. Looks like we mostly use underscores everywhere else so I would rather be consistent.